cloudigrade/config/settings/base.py

```diff
@@ -234,6 +234,11 @@
                                       default=AWS_NAME_PREFIX + \
                                               'inspection_results')
 
+CLOUDTRAIL_EVENT_URL = env(
```

Comment:
should we document what to set this environment variable to somewhere?

Reply:
Cloudigrade and Shiftigrade readmes have been updated.